doc/python_reference.rst

```diff
@@ -534,6 +534,14 @@ Events
    combine_event_ids
    equalize_epoch_counts
 
+Classes:
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   ElektaAverager
```

Comment:
I think the correct place for this is in the ``io`` section on line 108, below ``Raw``. Like this it does not link correctly, see https://2443-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/python_reference.html